src/utils.rs

```diff
@@ -57,3 +57,11 @@ pub(crate) fn jwt_payload_decode(b64data: &str) -> Result<serde_json::Map<String
     )
         .map_err(|e| DeserializationError(e.to_string()))
 }
+
+pub(crate) fn wrap_sd_key(key: &str) -> String {
+    if key.parse::<u64>().is_ok() {
```

Comment:
Seems like an expensive check. Can we wrap when we already know if the key is an array index instead? I'm referring to the `create_sd_claims_list` function.